src/number_theory/primorial.lean

```diff
@@ -47,12 +47,13 @@ begin
   have m_size : m + 1 ≤ 2 * m + 1 := le_of_lt (lt_of_lt_of_le p_big p_small),
   have s : ¬(p ∣ (m + 1)!),
   { intros p_div_fact,
-    have p_le_succ_m : p ≤ m + 1 := (prime.dvd_factorial is_prime).mp p_div_fact,
-    linarith, },
+    refine lt_le_antisymm p_big _,
+    exact is_prime.dvd_factorial.mp p_div_fact },
```

Comment:
```suggestion
    exact lt_le_antisymm p_big (is_prime.dvd_factorial.mp p_div_fact), },
```
untested! I guess there might be dot-notation for `lt_le_antisymm` too if you want